src/extent.c

```diff
@@ -2279,3 +2279,42 @@ extent_boot(void) {
 
 	return false;
 }
+
+void
+extent_util_stats_get(tsdn_t *tsdn, const void *ptr, size_t *nfree,
+    size_t *nregs, size_t *size, void **slabcur_addr) {
+	*nfree = *nregs = *size = 0;
```

Comment:
nit: We try avoiding double initialization, mainly for the purpose of explicitly setting the values, instead of relying on the default value to cover multiple cases (more error prone). Let's remove this line and the branch below, and see the comments below for changes.